descriptors/ModuleDescriptor-template.json

```
@@ -220,6 +220,19 @@
             "circulation-storage.loans.item.delete"
           ]
         },
+        {
+          "methods": [
+            "PUT"
+          ],
+          "pathPattern": "/circulation/loans/{id}/declare-lost",
```

As discussed in other changes, it is the item that is declared lost rather than the loan. I think the current path implies it is the loan that is declared lost, and suggest this be made more explicit. 

(if this change is made, please also reflect that in any permissions changes)

```suggestion
          "pathPattern": "/circulation/loans/{id}/declare-item-lost",
```

---

As I understand, you are suggesting to separate this new interface. Do you think it should be moved out of /circulation?

---

I’m not suggesting the path needs to be changed (beyond what I’ve suggested above), rather that it be defined as a separate interface

---

Done.